python/shotgun_model/shotgun_model.py

```diff
@@ -440,6 +440,7 @@ def _load_data(
                                           each column in the row contains the value for the corresponding field from columns. This means
                                           that the data from the loaded entity will be available field by field. Subclasses can modify
                                           this behavior by overriding _get_additional_columns.
+        :param editable_columns:          A subset of ``columns`` that will be editable in views that use this model.
```

Comment:
catching up here but what's the design thinking around putting this in the model vs controlling editability in the view? We often have multiple views hooked up to a single model and there are definitely cases where you want to make one view editable and one not. I think it would be great to just briefly address this use case in the doc string. Also if you could add some type info and maybe a usage example to both the columns and editable_columns param that would be great!